field-agent-ui/src/components/FieldAgent.js

```diff
@@ -0,0 +1,283 @@
+                      </div>
+
+                      <div className="field">
+                          <label htmlFor="dob">Date of Birth: </label>
+                          <input type="text" id="dob" name="dob" value={dob} onChange={dobNameOnChangeHandler} />
+                      </div>
+
+                      <div className="field">
+                          <label htmlFor="heightInInches">Height in Inches: </label>
+                          <input type="text" id="heightInInches" name="heightInInches" value={heightInInches} onChange={heightOnChangeHandler} />
+                      </div>
+
+                      <div className="field">
+                          <button type="submit">Add Agent</button>
+                          <button type="button" onClick={cancelClickHandler}>Cancel</button>
+                      </div>
+                  </form>
+              </div>
+          ) : null}
+
+          { currentView === 'Edit' ? (
+              <div>
+                  <h3>Edit Agent</h3>
+                  <form onSubmit={editAgentFormSubmitHandler}>
+                      <div className="field">
+                          <label htmlFor="firstName">First Name: </label>
+                          <input type="text" id="firstName" name="firstName" value={firstName} onChange={firstNameOnChangeHandler} />
+                      </div>
+
+                      <div className="field">
+                          <label htmlFor="middleName">Middle Name: </label>
+                          <input type="text" id="middleName" name="middleName" value={middleName} onChange={middleNameOnChangeHandler} />
+                      </div>
+
+                      <div className="field">
+                          <label htmlFor="lastName">Last Name: </label>
+                          <input type="text" id="lastName" name="lastName" value={lastName} onChange={lastNameOnChangeHandler} />
+                      </div>
+
+                      <div className="field">
+                          <label htmlFor="dob">Date of Birth: </label>
+                          <input type="text" id="dob" name="dob" value={dob} onChange={dobNameOnChangeHandler} />
+                      </div>
+
+                      <div className="field">
+                          <label htmlFor="heightInInches">Height in Inches: </label>
+                          <input type="text" id="heightInInches" name="heightInInches" value={heightInInches} onChange={heightOnChangeHandler} />
+                      </div>
+
+                      <div className="field">
+                          <button type="submit">Update Agent</button>
+                          <button type="button" onClick={cancelClickHandler}>Cancel</button>
+                      </div>
+                  </form>
+              </div>
+          ) : null}
+
+          {currentView === 'List' ? (
+              <div>
```

Comment:
Check to see where this `<div>` tag lives in relation to the `<div>`/container that encloses your cards. This could be the path back to your CSS issue.